src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CcdCaseUpdater.java

```diff
@@ -276,63 +291,26 @@ private String getErrorMessage(String service, String existingCaseId, String exc
         CaseDetails existingCase = startEvent.getCaseDetails();
 
         final CaseDataContent caseDataContent = getCaseDataContent(exceptionRecord, caseUpdateDetails, startEvent);
-        try {
-            coreCaseDataApi.submitEventForCaseWorker(
-                idamToken,
-                s2sToken,
-                userId,
-                exceptionRecord.poBoxJurisdiction,
-                startEvent.getCaseDetails().getCaseTypeId(),
-                existingCaseId,
-                ignoreWarnings,
-                caseDataContent
-            );
 
-            log.info(
-                "Successfully updated case for service {} "
-                    + "with case Id {} "
-                    + "based on exception record ref {}",
-                service,
-                existingCase.getId(),
-                exceptionRecord.id
-            );
-
-            return Optional.empty();
-        } catch (FeignException.UnprocessableEntity exception) {
-            String msg = format(
-                "CCD returned 422 Unprocessable Entity response "
-                    + "when trying to update case for %s jurisdiction "
-                    + "with case Id %s "
-                    + "based on exception record with Id %s. "
-                    + "CCD response: %s",
-                exceptionRecord.poBoxJurisdiction,
-                existingCase.getId(),
-                exceptionRecord.id,
-                exception.contentUTF8()
-            );
-            log.error(msg, exception);
-
-            return Optional.of(msg);
-        } catch (FeignException.Conflict exception) {
-            throw exception;
-        } catch (FeignException exception) {
```

Comment:
Not catching this exception here and retrowing RuntimeException changes the workflow because in the calling method where FeignException is caught it is not rethrown

Reply:
in the parent method conflict is caught and dealt with. rethrowing runtime exception is not great. it goes to general exception and throws callback exception. that is also dealt with

Reply:
Handling FeignException in the calling method (`updateCase`) is for exceptions thrown by the service (I assume line 113 in master branch), it is clearly stated in the log message _Failed to call 'updateCase'_. RuntimeException in the existing code be handled by the catch clause for generic exceptions.

Reply:
which should not be done in the first place ;)

Reply:
yes. but this PR should preserve existing difference in handling FeignExceptions thrown by calls to the service and to CCD

Reply:
hence there's a change in `bad request` handling and in general - a review of exception catches. that's the main focus here